packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -1184,11 +1184,13 @@ class _LineItemState extends State<LineItem>
     final column = widget.pausedFrame?.column;
     if (column != null) {
       final breakpointColor = theme.colorScheme.breakpointColor;
-
+      final widthToCurrentColumn = calculateTextSpanWidth(
+        truncateTextSpan(widget.lineContents, column - 1),
+      );
       // The following constants are tweaked for using the
       // 'Icons.label_important' icon.
       const colIconSize = 13.0;
-      const colLeftOffset = -3.0;
+      final colLeftOffset = -3.0 + widthToCurrentColumn;
```

Comment:
nit: change to `widthToCurrentColumn - 3.0` (if the 3 is still needed here? a comment could help)